src/dune/opam_create.ml

```diff
@@ -26,14 +26,30 @@ let default_build_command =
       "@doc" {with-doc}
   ]
 ]
+|}))
+  and from_2_07 =
```

Comment:
shouldn't this be `from_2_70`?

Reply:
I could rename it to `from_2_7` if that's too confusing.

Reply:
Yeah, `2_7` makes more sense to me. Otherwise, it reads like from `2.07`.

Reply:
Done.